semantics/src/lib.rs

```diff
@@ -106,8 +106,8 @@ impl Context {
     }
 
     /// Get information that has been attributed to an expression node.
-    pub fn get_expression(&self, spanned: &Spanned<fe::Expr>) -> Option<&ExpressionAttributes> {
-        self.expressions.get(&spanned.span)
+    pub fn get_expression<T: Into<Span>>(&self, span: T) -> Option<&ExpressionAttributes> {
```

Comment:
These getters previously worked with `Spanned<T>` but given the fact that they only actually care about the `Span` it was a bit unfortunate that one could not perform a lookup with a bare `Span`. So now they accept `Into<Span>` which means all the previous lookups still work as they did before but one can also use a bare `Span` to perform a lookup.